kore/src/Kore/Internal/TermLike.hs

```diff
@@ -2209,6 +2243,11 @@ pattern App_ applicationSymbolOrAlias applicationChildren <-
 pattern Bottom_ bottomSort <-
     (Recursive.project -> _ :< BottomF Bottom { bottomSort })
 
+pattern Bytes_ bytesSort string2BytesSymbol bytesValue <-
```

Comment:
I would prefer to use `InternalBytes` everywhere.

Reply:
Does that include module names like `Kore/Builtin/InternalBytes` instead of `Bytes`?

Reply:
Yes. I think it's better to have the module name match the type name.